src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,15 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+
+        Ball ball1 = lottery.getRandomBall();
+        Ball ball2 = lottery.getRandomBall();
+        Ball ball3 = lottery.getRandomBall();
```

Comment:
Consider using a loop to create multiple `Ball` objects. According to the checklist, using a loop is a better practice when creating several instances of the same class. This will make your code cleaner and more scalable.

Reply:
Fixed in [#d85f4de](https://github.com/mate-academy/jv-lottery/pull/1733/commits/d85f4dea1b7e4877c5ecb77f50fd1f9a33af49c0)